### ISSUE TITLE
Deposit Option In Funds Page Does Not Load Bitcoin Address

### ISSUE THREAD

**Issue by ShivSF006** [InfraexDev/peatio#64]
Deposit Option In Funds Page Does Not Load Bitcoin Address - 
I've Set RPC user & Password in bitcoin.conf and even in currency.yml with same details .. but still its not replicating.. Looking for a help ASAP

**Comment by EliteMobb79** [comment]
I am having the same issue. Same with ETH as well.

**Comment by nantestdevoloper** [comment]
yeah I am able to generate it. Please check your RAM and memory size as well. This can be one of the reason

**Comment by archana209** [comment]
Same problem with me as well. How to debug ? bitcoin-conf and currency.yml have same info

**Comment by muhammednagy** [comment]
hello guys
sorry for the late reply
it's working
here are some rules for it to work though
1. don't use the same server for bitcoin & ethereum & peatio
2. make sure no firewall is blocking the connections
3.make sure the username and the password are correct

**Comment by shouton** [comment]
@muhammednagy

Agree with isolating the bitcoind with the main server for security reason. 

But, is it must? I mean in development mode, it must be located in another server?

**Comment by muhammednagy** [comment]
no
anyways the issue with not generating should be that you need to allow your localhost ip
like 127.0.0.1
